examples/api-routes-rest/README.md

```diff
@@ -0,0 +1,42 @@
+# API routes with REST
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/segmentio/create-next-app) with [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) or [npx](https://github.com/zkat/npx#readme) to bootstrap the example:
+
+```bash
+npx create-next-app --example api-routes-rest api-routes-rest-app
+# or
+yarn create next-app --example api-routes-rest api-routes-rest-app
+```
+
+### Download manually
```

Comment:
Let's drop all these "Download manually" helpers from examples. Just remove it from this PR for now, we can make that update to all in the future.